templates/repo/commits_list.tmpl

```diff
@@ -76,6 +77,11 @@
 						{{else}}
 							<td class="text right aligned">{{TimeSince .Author.When ctx.Locale}}</td>
 						{{end}}
+						{{if $.FileName}}
+							<td class="text right aligned">
+								<a class="ui basic button copy-commit-sha" href="{{printf "%s/src/commit/%s/%s" $commitRepoLink (PathEscape .ID.String) $.FileName}}">{{svg "octicon-file-code"}}</a>
```

Comment:
```suggestion
								<a class="btn interact-bg copy-commit-sha" href="{{printf "%s/src/commit/%s/%s" $commitRepoLink (PathEscape .ID.String) $.FileName}}">{{svg "octicon-file-code"}}</a>
```

Should look better without border/bg and it's consistent with branch list buttons.